beacon-chain/sync/rpc_ping.go

```diff
@@ -101,11 +81,7 @@ func (s *Service) sendPingRequest(ctx context.Context, id peer.ID) error {
 		return err
 	}
 	currentTime := timeutils.Now()
-	defer func() {
-		if err := helpers.FullClose(stream); err != nil && err.Error() != mux.ErrReset.Error() {
-			log.WithError(err).Debugf("Could not reset stream with protocol %s", stream.Protocol())
-		}
-	}()
+	defer closeStream(stream, log)
```

Comment:
note: I never bother calling Reset after Send because Send closes for writing. There are no writes to abort at this point.